### DISCUSSION TITLE
License for ethical use of software

### DISCUSSION THREAD

**Discussion by roadkell** [Tyrrrz/DiscordChatExporter#1014] [General]
Hello! You might be interested in a license that is aimed at restricting the possible uses of software to ethical only means (in contrary to GPL/LGPL and such): [Hippocratic License](https://firstdonoharm.dev/). It is based on the Universal Declaration of Human Rights and may be useful to convey the anti-war message, in addition to ToU.
Stay safe!

**Comment by Tyrrrz** [comment]
Thanks for the hint. I do like the implications enforced by GPL though, is it possible to use one license as extension to another?

**Reply by roadkell** [reply]
HL itself is incompatible with GPL and other open source licenses, as it imposes restrictions on possible uses of software ([the discussion here](https://github.com/EthicalSource/hippocratic-license/issues/6) has more info, for example), but it also has a [copyleft module](https://firstdonoharm.dev/build/?modules=cl#8.3) which requires derivative works to use the same license, similarly to strong copyleft licenses like GPL.
